app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -122,7 +122,7 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
 
     private fun setReportButton() {
 
-        if(PrefManager.getToken().isNotEmpty()){
+        if(PrefManager.getToken() != null){
```

Comment:
Can you please also added spaces around parenthesis. Opportunities like this allow up to improve the code bit by bit :)

Reply:
@iamareebjamal Thanks for pointing this out.
I should have done it before sending the PR. Updating it. 🙂